flows/contact.go

```diff
@@ -526,8 +530,7 @@ func ReadContact(sa SessionAssets, data json.RawMessage, missing assets.MissingC
 		id:        envelope.ID,
 		name:      envelope.Name,
 		language:  envelope.Language,
-		blocked:   envelope.Blocked,
-		stopped:   envelope.Stopped,
+		status:    envelope.Status,
```

Comment:
one last tiny tweak - need to default `status` to `active` if it's not set in the JSON